recipes/rsirius/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.9" %}
+{% set name = "r-sirius-ms" %}
+{% set commit = "d1274f51b2af44c09def72fa3628f82a9e8de6b5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/boecker-lab/sirius-client-openAPI/archive/{{ commit }}.zip
+  sha256: 1edef52c472b9984b74532cf41ae9389f4191579085fd981922d8356d0ba887a
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  build:
+    - r-base >=3.5.0
+    - r-jsonlite
+    - r-httr2
+    - r-r6
+    - r-base64enc
+    - r-stringr
+  run:
+    - r-base >=3.5.0
+    - r-jsonlite
+    - r-httr2
+    - r-r6
+    - r-base64enc
+    - r-stringr
+    - sirius-ms
+
+test:
+  commands:
```

Comment:
Is it necessary to explicitly run these files? At least according to the [docs](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section), they should be automatically detected and executed:

> if there is a `run_test.[py,pl,sh,bat]` file in the recipe, the package is installed into a test environment after the build is finished and the tests are run there.